2023-07-10 18:26:19,438 INFO    StreamThr :15734 [internal.py:wandb_internal():86] W&B internal server running at pid: 15734, started at: 2023-07-10 18:26:19.437919
2023-07-10 18:26:19,440 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status
2023-07-10 18:26:19,441 INFO    WriterThread:15734 [datastore.py:open_for_write():85] open: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_182619-tk9ohpoz/run-tk9ohpoz.wandb
2023-07-10 18:26:19,443 DEBUG   SenderThread:15734 [sender.py:send():369] send: header
2023-07-10 18:26:19,443 DEBUG   SenderThread:15734 [sender.py:send():369] send: run
2023-07-10 18:26:20,406 INFO    SenderThread:15734 [dir_watcher.py:__init__():211] watching files in: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_182619-tk9ohpoz/files
2023-07-10 18:26:20,406 INFO    SenderThread:15734 [sender.py:_start_run_threads():1100] run started: tk9ohpoz with start time 1688984779.43733
2023-07-10 18:26:20,407 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: summary_record
2023-07-10 18:26:20,407 INFO    SenderThread:15734 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-07-10 18:26:20,413 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: check_version
2023-07-10 18:26:20,413 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: check_version
2023-07-10 18:26:20,593 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: run_start
2023-07-10 18:26:20,597 DEBUG   HandlerThread:15734 [system_info.py:__init__():31] System info init
2023-07-10 18:26:20,598 DEBUG   HandlerThread:15734 [system_info.py:__init__():46] System info init done
2023-07-10 18:26:20,598 INFO    HandlerThread:15734 [system_monitor.py:start():181] Starting system monitor
2023-07-10 18:26:20,598 INFO    HandlerThread:15734 [system_monitor.py:probe():201] Collecting system info
2023-07-10 18:26:20,599 INFO    SystemMonitor:15734 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-07-10 18:26:20,600 INFO    SystemMonitor:15734 [interfaces.py:start():190] Started cpu monitoring
2023-07-10 18:26:20,601 INFO    SystemMonitor:15734 [interfaces.py:start():190] Started disk monitoring
2023-07-10 18:26:20,602 INFO    SystemMonitor:15734 [interfaces.py:start():190] Started gpu monitoring
2023-07-10 18:26:20,603 INFO    SystemMonitor:15734 [interfaces.py:start():190] Started memory monitoring
2023-07-10 18:26:20,603 INFO    SystemMonitor:15734 [interfaces.py:start():190] Started network monitoring
2023-07-10 18:26:20,617 DEBUG   HandlerThread:15734 [system_info.py:probe():195] Probing system
2023-07-10 18:26:20,621 DEBUG   HandlerThread:15734 [gitlib.py:_init_repo():53] git repository is invalid
2023-07-10 18:26:20,621 DEBUG   HandlerThread:15734 [system_info.py:probe():240] Probing system done
2023-07-10 18:26:20,621 DEBUG   HandlerThread:15734 [system_monitor.py:probe():210] {'os': 'Linux-5.19.0-46-generic-x86_64-with-glibc2.35', 'python': '3.9.16', 'heartbeatAt': '2023-07-10T10:26:20.618031', 'startedAt': '2023-07-10T10:26:19.429595', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/home/zhuhe/HPE-with-Diffusion/main.py', 'codePath': 'main.py', 'host': '415-2', 'username': 'zhuhe', 'executable': '/home/zhuhe/.conda/envs/diffhpe/bin/python', 'cpu_count': 16, 'cpu_count_logical': 24, 'cpu_freq': {'current': 3475.1787083333334, 'min': 800.0, 'max': 4783.333333333333}, 'cpu_freq_per_core': [{'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 4710.733, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5099.865, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 2100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 4206.278, 'min': 800.0, 'max': 5100.0}, {'current': 5099.987, 'min': 800.0, 'max': 5200.0}, {'current': 2100.0, 'min': 800.0, 'max': 5200.0}, {'current': 5100.0, 'min': 800.0, 'max': 5200.0}, {'current': 2100.0, 'min': 800.0, 'max': 5200.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 2100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 3699.559, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}], 'disk': {'total': 93.31179428100586, 'used': 0.01779937744140625}, 'gpu': 'NVIDIA GeForce RTX 3090', 'gpu_count': 2, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}, {'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}], 'memory': {'total': 31.185558319091797}}
2023-07-10 18:26:20,621 INFO    HandlerThread:15734 [system_monitor.py:probe():211] Finished collecting system info
2023-07-10 18:26:20,621 INFO    HandlerThread:15734 [system_monitor.py:probe():214] Publishing system info
2023-07-10 18:26:20,621 DEBUG   HandlerThread:15734 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-07-10 18:26:20,622 DEBUG   HandlerThread:15734 [system_info.py:_save_pip():67] Saving pip packages done
2023-07-10 18:26:20,622 DEBUG   HandlerThread:15734 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-07-10 18:26:21,383 DEBUG   HandlerThread:15734 [system_info.py:_save_conda():86] Saving conda packages done
2023-07-10 18:26:21,385 INFO    HandlerThread:15734 [system_monitor.py:probe():216] Finished publishing system info
2023-07-10 18:26:21,387 DEBUG   SenderThread:15734 [sender.py:send():369] send: files
2023-07-10 18:26:21,387 INFO    SenderThread:15734 [sender.py:_save_file():1354] saving file wandb-metadata.json with policy now
2023-07-10 18:26:21,393 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 18:26:21,394 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: stop_status
2023-07-10 18:26:21,408 INFO    Thread-17 :15734 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_182619-tk9ohpoz/files/wandb-metadata.json
2023-07-10 18:26:21,409 INFO    Thread-17 :15734 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_182619-tk9ohpoz/files/wandb-summary.json
2023-07-10 18:26:21,409 INFO    Thread-17 :15734 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_182619-tk9ohpoz/files/requirements.txt
2023-07-10 18:26:21,409 INFO    Thread-17 :15734 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_182619-tk9ohpoz/files/conda-environment.yaml
2023-07-10 18:26:21,937 ERROR   wandb-upload_0:15734 [internal_api.py:upload_file():2192] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/tk9ohpoz/wandb-metadata.json?Expires=1689071181&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=W6pMK1F2OokM%2BS7xXHFFr1F9KTkTtiDa6WzGT2XydBp9bEIGLMrSexqe2tZoV0oQ9B4vkXB%2BNz4RGbfFLaG%2B04NsW4wE5oVPhpu%2B15gZU2S16yL1lTvewM4Rp%2FOenG%2FBA3Oo4XeQB1inpAhnH2EO3DAza50fgYKMIqKe0gvLl1tgWCpkplyO2aeglRLOcSBSXtSMmYCQyFlYJdcdFySQIwEO5CsqbZubNRs%2BZoqzgPBDDbcfdbwtKY2CuZ%2BMB3N011iiIdq7CHJSw%2BIqfpUbyWRkWYPKS9zpSaUxvgF2SPsrlUP%2FQy12su9LLnPwBZH9dnDkdlSqcptXG67ebK11TQ%3D%3D: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/tk9ohpoz/wandb-metadata.json?Expires=1689071181&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=W6pMK1F2OokM%2BS7xXHFFr1F9KTkTtiDa6WzGT2XydBp9bEIGLMrSexqe2tZoV0oQ9B4vkXB%2BNz4RGbfFLaG%2B04NsW4wE5oVPhpu%2B15gZU2S16yL1lTvewM4Rp%2FOenG%2FBA3Oo4XeQB1inpAhnH2EO3DAza50fgYKMIqKe0gvLl1tgWCpkplyO2aeglRLOcSBSXtSMmYCQyFlYJdcdFySQIwEO5CsqbZubNRs%2BZoqzgPBDDbcfdbwtKY2CuZ%2BMB3N011iiIdq7CHJSw%2BIqfpUbyWRkWYPKS9zpSaUxvgF2SPsrlUP%2FQy12su9LLnPwBZH9dnDkdlSqcptXG67ebK11TQ%3D%3D
2023-07-10 18:26:21,937 ERROR   wandb-upload_0:15734 [internal_api.py:upload_file():2194] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3615'}
2023-07-10 18:26:21,937 ERROR   wandb-upload_0:15734 [internal_api.py:upload_file():2196] upload_file response body: b'<!DOCTYPE html>\n<html lang=en>\n  <meta charset=utf-8>\n  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">\n  <title>Error 404 (Not Found)!!1</title>\n  <style>\n    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}\n  </style>\n  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>\n  <p><b>404.</b> <ins>That\xe2\x80\x99s an error.</ins>\n  <p>The requested URL <code>/wandb-production.appspot.com/ada4321/DiffHPE/tk9ohpoz/wandb-metadata.json</code> was not found on this server.  <ins>That\xe2\x80\x99s all we know.</ins>\n'
2023-07-10 18:26:21,996 DEBUG   SenderThread:15734 [sender.py:send():369] send: telemetry
2023-07-10 18:26:22,407 INFO    Thread-17 :15734 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_182619-tk9ohpoz/files/output.log
2023-07-10 18:26:22,571 ERROR   wandb-upload_0:15734 [upload_job.py:push():136] Failed to upload file: /tmp/tmpjn2v6msrwandb/dv50o6us-wandb-metadata.json
Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 212, in wrapped_fn
    return retrier(*args, **kargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2218, in upload_file
    wandb._sentry.reraise(e)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/analytics/sentry.py", line 146, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2190, in upload_file
    response.raise_for_status()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/tk9ohpoz/wandb-metadata.json?Expires=1689071181&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=W6pMK1F2OokM%2BS7xXHFFr1F9KTkTtiDa6WzGT2XydBp9bEIGLMrSexqe2tZoV0oQ9B4vkXB%2BNz4RGbfFLaG%2B04NsW4wE5oVPhpu%2B15gZU2S16yL1lTvewM4Rp%2FOenG%2FBA3Oo4XeQB1inpAhnH2EO3DAza50fgYKMIqKe0gvLl1tgWCpkplyO2aeglRLOcSBSXtSMmYCQyFlYJdcdFySQIwEO5CsqbZubNRs%2BZoqzgPBDDbcfdbwtKY2CuZ%2BMB3N011iiIdq7CHJSw%2BIqfpUbyWRkWYPKS9zpSaUxvgF2SPsrlUP%2FQy12su9LLnPwBZH9dnDkdlSqcptXG67ebK11TQ%3D%3D

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 127, in push
    self._api.upload_file_retry(
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: <Response [404]>
2023-07-10 18:26:22,789 ERROR   Thread-16 :15734 [step_upload.py:_handle_event():200] Failed to upload file: /tmp/tmpjn2v6msrwandb/dv50o6us-wandb-metadata.json
Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 212, in wrapped_fn
    return retrier(*args, **kargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2218, in upload_file
    wandb._sentry.reraise(e)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/analytics/sentry.py", line 146, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2190, in upload_file
    response.raise_for_status()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/tk9ohpoz/wandb-metadata.json?Expires=1689071181&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=W6pMK1F2OokM%2BS7xXHFFr1F9KTkTtiDa6WzGT2XydBp9bEIGLMrSexqe2tZoV0oQ9B4vkXB%2BNz4RGbfFLaG%2B04NsW4wE5oVPhpu%2B15gZU2S16yL1lTvewM4Rp%2FOenG%2FBA3Oo4XeQB1inpAhnH2EO3DAza50fgYKMIqKe0gvLl1tgWCpkplyO2aeglRLOcSBSXtSMmYCQyFlYJdcdFySQIwEO5CsqbZubNRs%2BZoqzgPBDDbcfdbwtKY2CuZ%2BMB3N011iiIdq7CHJSw%2BIqfpUbyWRkWYPKS9zpSaUxvgF2SPsrlUP%2FQy12su9LLnPwBZH9dnDkdlSqcptXG67ebK11TQ%3D%3D

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 127, in push
    self._api.upload_file_retry(
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: <Response [404]>
2023-07-10 18:26:24,407 INFO    Thread-17 :15734 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_182619-tk9ohpoz/files/output.log
2023-07-10 18:26:25,000 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:26:30,000 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:26:35,001 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:26:36,392 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 18:26:36,392 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: stop_status
2023-07-10 18:26:40,635 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:26:45,636 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:26:50,643 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:26:51,392 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 18:26:51,392 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: stop_status
2023-07-10 18:26:51,415 INFO    Thread-17 :15734 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_182619-tk9ohpoz/files/config.yaml
2023-07-10 18:26:56,617 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:01,617 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:06,392 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 18:27:06,393 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: stop_status
2023-07-10 18:27:06,625 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:11,626 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:16,627 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:20,604 DEBUG   SystemMonitor:15734 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-07-10 18:27:20,605 DEBUG   SenderThread:15734 [sender.py:send():369] send: stats
2023-07-10 18:27:21,392 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 18:27:21,393 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: stop_status
2023-07-10 18:27:22,621 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:27,622 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:32,623 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:36,392 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 18:27:36,393 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: stop_status
2023-07-10 18:27:37,624 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:42,625 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:47,625 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:50,606 DEBUG   SenderThread:15734 [sender.py:send():369] send: stats
2023-07-10 18:27:51,393 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 18:27:51,394 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: stop_status
2023-07-10 18:27:53,617 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:27:58,618 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:28:03,618 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:28:06,393 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 18:28:06,394 DEBUG   SenderThread:15734 [sender.py:send_request():396] send_request: stop_status
2023-07-10 18:28:08,629 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 18:28:13,630 DEBUG   HandlerThread:15734 [handler.py:handle_request():144] handle_request: status_report
